### DISCUSSION TITLE
When the response packet has no header information, an error message is triggered.

### DISCUSSION THREAD

**Discussion by c0mpl1ng** [projectdiscovery/nuclei#5265] [Q&A]
When some poc response packages do not have a response header, the following error is raised.

<img width="740" alt="image" src="https://github.com/projectdiscovery/nuclei/assets/52808596/baafd4d4-5677-4164-a38f-0f32621a4305">

The response package is as follows:
<img width="719" alt="image" src="https://github.com/projectdiscovery/nuclei/assets/52808596/04b76d74-8e35-4795-863e-f106a8eef9ac">

**Comment by tarunKoyalwar** [comment]
@King-Mr , as of now we only support `unsafe` or raw request , and from the screenshot you have shared it is not a valid response as per RFCs , so go stdlib is unable to parse the response , it might be good idea to use achieve the same using javascript protocol where you don't parse the response and directly match on output data 

see: https://github.com/projectdiscovery/nuclei/blob/dev/integration_tests/protocols/javascript/net-https.yaml

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
